src/develop/imageop.c

```diff
@@ -174,6 +176,19 @@ static int default_distort_backtransform(dt_iop_module_t *self, dt_dev_pixelpipe
   return 1;
 }
 
+static void default_process(struct dt_iop_module_t *self, struct dt_dev_pixelpipe_iop_t *piece, void *i,
+                            void *o, const struct dt_iop_roi_t *roi_in, const struct dt_iop_roi_t *roi_out)
+{
+  if(darktable.codepath.OPENMP_SIMD && self->process_plain)
+    self->process_plain(self, piece, i, o, roi_in, roi_out);
```

Comment:
so you did rename the innocent process() to process_plain() after all? why? to make sure it doesn't use SSE and force us to look over every single module?